mock/py/mockbuild/backend.py

```diff
@@ -119,34 +121,24 @@ def scrub(self, scrub_opts):
                     elif scrub == 'chroot':
                         self.buildroot.root_log.info("scrubbing chroot for %s", self.config_name)
                         self.buildroot.delete()
-                        if self.bootstrap_buildroot is not None:
-                            self.bootstrap_buildroot.delete()
                     elif scrub == 'cache':
                         self.buildroot.root_log.info("scrubbing cache for %s", self.config_name)
                         util.rmtree(self.buildroot.cachedir, selinux=self.buildroot.selinux)
-                        if self.bootstrap_buildroot is not None:
-                            util.rmtree(self.bootstrap_buildroot.cachedir, selinux=self.bootstrap_buildroot.selinux)
                     elif scrub == 'c-cache':
                         self.buildroot.root_log.info("scrubbing c-cache for %s", self.config_name)
                         util.rmtree(os.path.join(self.buildroot.cachedir, 'ccache'), selinux=self.buildroot.selinux)
-                        if self.bootstrap_buildroot is not None:
-                            util.rmtree(os.path.join(self.bootstrap_buildroot.cachedir, 'ccache'),
-                                        selinux=self.bootstrap_buildroot.selinux)
                     elif scrub == 'root-cache':
                         self.buildroot.root_log.info("scrubbing root-cache for %s", self.config_name)
                         util.rmtree(os.path.join(self.buildroot.cachedir, 'root_cache'), selinux=self.buildroot.selinux)
-                        if self.bootstrap_buildroot is not None:
-                            util.rmtree(os.path.join(self.bootstrap_buildroot.cachedir, 'root_cache'),
-                                        selinux=self.bootstrap_buildroot.selinux)
```

Comment:
I do not get why these lines are being removed for root-cache, c-cache, cache, and yum-cache and dnf-cache?

Reply:
That's what we think is actually desired;  people doing `--scrub=root-cache` unlikely want to also clean `root-cache` of bootstrap chroot (they shouldn't mind, or even shouldn't basically know there's bootstrap chroot once it is default).

Reply:
But if they know and they want to delete it, then there will be no way

Reply:
Right, they'd have to do `--scrub=bootstrap`, but yes ... most valid would be to have `--scrub=bootstrap-root-cache`.

Reply:
IIRC, the `--scrub=bootstrap` will not clean bootstrap root cache